examples/api/mathtext_asarray.py

```diff
@@ -22,7 +22,7 @@
     r'some other string', color='red', fontsize=20, dpi=200)
 
 fig = plt.figure()
-fig.figimage(rgba1.astype(float)/255., 100, 100)
-fig.figimage(rgba2.astype(float)/255., 100, 300)
+fig.figimage(rgba1, 100, 100)
```

Comment:
Why did you remove the division by 255?

Reply:
because we can display uint8 images just as well as float images, and the real point of the example is to show how to get the mathtext as an image array...